tasks/__init__.py

```diff
@@ -504,6 +519,10 @@ def test_travis_else():
     karma(single=True, browsers='PhantomJS')
 
 @task
+def test_travis_varnish():
```

Comment:
Oh, but I just noticed that the varnish tests run as a separate parallel build. I see what you mean now. OK, I think this fine either way.